solarforecastarbiter/reports/tests/test_reports_main.py

```diff
@@ -70,6 +71,30 @@ def test_full_render(mock_data, report_objects):
         f.write(full_report)
 
 
+@pytest.mark.skipif(shutil.which('pandoc') is None,
+                    reason='Pandoc can not be found')
+def test_all_categories_render(mock_data, report_objects):
+    report, observation, forecast_0, forecast_1 = report_objects
+    all_report = datamodel.Report(
+        name=report.name,
+        start=report.start,
+        end=report.end,
+        forecast_observations=report.forecast_observations,
+        metrics=("mae", "rmse", "mbe"),
+        categories=list(calculator.AVAILABLE_CATEGORIES.keys()),
+        report_id=report.report_id,
+        filters=report.filters
+    )
+    session = api.APISession('nope')
+    data = main.get_data_for_report(session, all_report)
+    raw_report = main.create_raw_report_from_data(all_report, data)
+    report_md = main.render_raw_report(raw_report)
+    body = template.report_md_to_html(report_md)
+    full_report = template.full_html(body)
```

Comment:
should make sure the categories are actually in the <body> of full_report

Reply:
Yeah I make sure to run a report with all categories when I check it. We should probably do the same with the deterministic metrics if no one has already been checking that.